_data/current.yml

```diff
@@ -1,4 +1,11 @@
-- name: The Premier Online Conference For Software Testing
+- name: Elevating Quality: Expert Insights on QA & Software Testing
+  location: LinkedIn Live Event
+  dates: "June 06, 2024"
+  event link: https://www.linkedin.com/events/elevatingquality-expertinsights7199022994160050176/
+  url: https://bit.ly/3Kn0pcg
+  status: Registration is Open
+  
+  - name: The Premier Online Conference For Software Testing
```

Comment:
The indentation of this line will cause the whole build to fail. You can't have spaces before the `-`.